cva6/regress/smoke-tests.sh

```diff
@@ -26,6 +26,7 @@ if ! [ -n "$DV_SIMULATORS" ]; then
 fi
 
 cd cva6/sim/
+python3 cva6.py --testlist=../tests/testlist_riscv-csr-access-test-cv32a60x.yaml --test MSCRATCH_CSR --iss_yaml cva6.yaml --target cv32a60x --iss=$DV_SIMULATORS $DV_OPTS  --linker=../sim/link.ld
```

Comment:
The CSR tests will be executed from dv-riscv-csr-access-test.sh, you do not need to modify the smoke_tests.sh

Reply:
Removed CSR access test from smoke-tests.sh